crates/iota-framework-tests/src/metered_verifier.rs

```diff
@@ -26,201 +26,205 @@ fn build(path: &Path) -> IotaResult<CompiledPackage> {
     config.build(path)
 }
 
-#[test]
-#[cfg_attr(msim, ignore)]
-fn test_metered_move_bytecode_verifier() {
-    move_package::package_hooks::register_package_hooks(Box::new(IotaPackageHooks));
-    let path =
-        PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("../iota-framework/packages/iota-framework");
-    let compiled_package = build(&path).unwrap();
-    let compiled_modules: Vec<_> = compiled_package.get_modules().cloned().collect();
-
-    let protocol_config = ProtocolConfig::get_for_max_version_UNSAFE();
-    let mut verifier_config = protocol_config.verifier_config(/* for_signing */ true);
-    let mut meter_config = protocol_config.meter_config_for_signing();
-    let registry = &Registry::new();
-    let bytecode_verifier_metrics = Arc::new(BytecodeVerifierMetrics::new(registry));
-    let mut meter = IotaVerifierMeter::new(meter_config.clone());
-    let timer_start = Instant::now();
-    // Default case should pass
-    let r = run_metered_move_bytecode_verifier(
-        &compiled_modules,
-        &verifier_config,
-        &mut meter,
-        &bytecode_verifier_metrics,
-    );
-    let elapsed = timer_start.elapsed().as_micros() as f64 / (1000.0 * 1000.0);
-    assert!(r.is_ok());
-
-    // Ensure metrics worked as expected
-
-    // The number of module success samples must equal the number of modules
-    assert_eq!(
-        compiled_modules.len() as u64,
-        bytecode_verifier_metrics
+mod move_tests {
```

Comment:
Was adding this here necessary? as it is iota-framework-tests that is already included in the CI rule. Or wrap all tests in iota-framework-tests with move_test and simplify condition

Reply:
You're probably right!

Reply:
Technically speaking, some of the tests in this file aren't necessary to re-run, but the currently test filter includes them regardless and I don't think it's worth separating them out, so I reverted this module.